README.md

```diff
@@ -1 +1,24 @@
 # Operator Marketplace
+
```

Comment:
Please add a note mentioning this is pre-alpha software

Reply:
Note added.